apps/src/templates/JsDebugger.jsx

```diff
@@ -217,13 +227,20 @@ var JsDebugger = React.createClass({
         </PaneHeader>
         {this.props.debugButtons && <DebugButtons/>}
         {this.props.debugConsole && <DebugConsole debugButtons={this.props.debugButtons} debugWatch={showWatchPane}/>}
-        {showWatchPane && <DebugWatch debugButtons={this.props.debugButtons}/>}
+        <div style={{display: showWatchPane ? 'initial' : 'none'}}>
+          <ProtectedStatefulDiv>
+            <div id="watchersResizeBar"></div>
+          </ProtectedStatefulDiv>
+          </div>
```

Comment:
It toggles the open/closed state of the watch area, the reason it's being done with style & with a `ProtectedStatefulDiv` is so `JsDebuggerUi.js`'s `watchersResizeBar` listener does not get removed.